package.json

```diff
@@ -124,6 +124,8 @@
     "debug": "~4.3.4",
     "eslint": "~8.48.0",
     "eslint-config-prettier": "~9.0.0",
+    "eslint-import-resolver-typescript": "^3.6.0",
+    "eslint-plugin-import": "^2.28.1",
```

Comment:
**nitpick:** use ~ instead of ^ for versions. Otherwise, new rules may generate errors locally without prior notice when running npm install.